---
vigir_footstep_planning_msgs/StepPlan step_plan
vigir_footstep_planning_msgs/ErrorStatus status
